Dockerfile

```diff
@@ -1,14 +1,30 @@
-FROM ruby:2.2.10-alpine
+FROM alpine:3.6
 
 WORKDIR /usr/src/app
 COPY Gemfile /usr/src/app/
 COPY Gemfile.lock /usr/src/app/
 COPY VERSION /usr/src/app/
 COPY codeclimate.gemspec /usr/src/app/
+ENV CODECLIMATE_DOCKER=1 BUNDLE_SILENCE_ROOT_WARNING=1
 
-RUN apk --update add git openssh-client wget build-base && \
-    bundle install -j 4 && \
-    apk del build-base && rm -fr /usr/share/ri
+RUN apk --no-cache upgrade && \
+      apk --no-cache --update add \
+      build-base \
+      ca-certificates \
+      git \
+      openssh-client \
+      openssl \
+      ruby \
+      ruby-bigdecimal \
+      ruby-bundler \
+      ruby-dev \
+      wget && \
+      bundle install -j 4 && \
+      apk del build-base && \
+      rm -fr /usr/share/ri && \
+      printf 'export PATH=$PATH:/usr/lib/ruby/gems/2.4.0/bin' \
```

Comment:
It seems like this will be a little more challenging to maintain, since we'll need to update this line whenever the apk registry switches over to 2.5 as the ruby that gets installed when someone runs `apk add ruby` :thinking:.

Users who use the CLI will pull the image, not build it themselves, so it's not likely that this occurrence will break things for users, but it could be a bit of a headache in the future for us. Do you agree? Your POV is that it's worth it for the sake of slimming the image, or are there other reasons as well?

Reply:
Actually, that part may not be necessary in the first place.
I added it because it's what the `codeclimate/docker-alpine-ruby` Dockerfile we previously inherited from was doing. (Though there the path was set to 2.2.0)

Let me remove that and run the checks again; it shouldn't actually be necessary, because we're happy with the default paths built into the binary package.